src/datasources/tag/TagDataSource.ts

```diff
@@ -25,27 +25,53 @@ export class TagDataSource extends DataSourceBase<TagQuery> {
   tagUrl = this.instanceSettings.url + '/nitag/v2';
   tagHistoryUrl = this.instanceSettings.url + '/nitaghistorian/v2/tags';
 
-  defaultQuery = {
+  defaultQuery: Omit<TagQuery, 'refId'> = {
     type: TagQueryType.Current,
     path: '',
     workspace: '',
     properties: false,
   };
 
   async runQuery(query: TagQuery, { range, maxDataPoints, scopedVars }: DataQueryRequest): Promise<DataFrameDTO> {
-    const { tag, current } = await this.getLastUpdatedTag(
+    const tagsLastUpdates: TagWithValue[] = await this.getLastUpdatedTag(
       this.templateSrv.replace(query.path, scopedVars),
       this.templateSrv.replace(query.workspace, scopedVars)
     );
 
+    const tag = tagsLastUpdates[0].tag
     const name = tag.properties?.displayName ?? tag.path;
```

Comment:
Move this to the history branch of `runQuery` since it's no longer used in the `query.type === TagQueryType.Current` branch

Reply:
```suggestion
    if (query.properties) {
      result.fields = result.fields.concat(this.getPropertiesAsFields(tag.properties));
    }
```

But used here, outside of query.type check branch
Since we divided PR into 4 parts, I will delete this part in the second PR when I work with history case and will remove ability to show properties with history